examples/jsonpath-expressions/jsonpath_expressions.md

```diff
@@ -0,0 +1,11 @@
+# JSONPath expressions
+
+JSONPath expressions provide a straightforward method for navigating the structure of JSON data.
+
```

Comment:
Wonder if in a separate para we should say "Alternatively, ..." and mention about Ballerina JSON access (linking to those examples), maybe saying that's strongly typed.

Reply:
updated